drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -28,7 +30,11 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   # A user can see any enrollment associated with a project they can access
   scope :viewable_by, ->(user) do
-    joins(:project).merge(Hmis::Hud::Project.viewable_by(user))
+    project_ids = Hmis::Hud::Project.viewable_by(user).pluck(:id, :ProjectID)
+    vieawable_wip = wip_t[:project_id].in(project_ids.map(&:first))
+    viewable_enrollment = e_t[:ProjectID].in(project_ids.map(&:second))
```

Comment:
@eanders I could use `ArelHelper` but it means I have to use `:ProjectID` instead of `:project_id` here.

Reply:
That's true.  I'll try to wake up the idea of allowing snake case for the warehouse fields again this week.